drake/solvers/NloptSolver.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DRAKE_NLOPT_SOLVER_H
```

Comment:
Oh, I see what's going on.  The errors are being suppressed by `drake/CPPLINT.cfg`.  I'll work on a minimally-annoying way to run cpplint on diffs without the repository-wide suppressions.  In the meantime, running cpplint is still a good idea for the stuff we aren't suppressing, but I'll stop pointing it out for the things that we are.   Sorry about that!